testing/gcin/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Contributor: Felix Yan <felixonmars@archlinux.org>
+# Contributor: Gaetan Bisson <bisson@archlinux.org>
+# Contributor: Andreas Radke <andyrtr@archlinux.org>
+# Contributor: damir <damir@archlinux.org>
+# above are from Arch Linux
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+
+pkgname=gcin
+pkgver=2.8.4
+pkgrel=0
+pkgdesc='Input method server supporting various input methods'
+url='http://hyperrate.com/dir.php?eid=67'
+license="LGPL"
+arch="all"
+#depends="gtk2 libxtst"
+depends="gtk+2.0 libxtst"
+#makedepends="qt5-base' 'qt4' 'gtk3' 'anthy' 'libchewing')
+makedepends="gtk+2.0-dev libxtst-dev qt5-qtbase-dev gtk+3.0-dev libchewing-dev"
+subpackages="$pkgname-doc"
+source="http://hyperrate.com/gcin-source/${pkgname}-${pkgver}.tar.xz
+  qt-5.5.patch qt5_inc.patch missing_u_int.patch"
+builddir="${srcdir}/${pkgname}-${pkgver}"
+install="gcin.post-install gcin.post-upgrade gcin.post-deinstall"
+
+prepare() {
+    cd "$builddir"
+    sed 's/lib64/lib/g' -i configure || return
+    sed '/include suffixes-rule/a \
+        CFLAGS+='"${CFLAGS}"' \
+        LDFLAGS+='"${LDFLAGS}"' \
+        OPTFLAGS=' \
+        -i Makefile || return
+    sed \
+        -e 's:usr/include/qt5:usr/include/qt:g' \
+        -e 's:QT=qt5:QT=qt:' \
+        -e '/^MODVERSION=/a INCS+=-I/usr/include/qt/QtGui/$(MODVERSION) -I/usr/include/qt/QtCore/$(MODVERSION)' \
+        -i qt5-im/Makefile || return
+
+    # FS#45732
+    default_prepare || return
+
+    # Qt 5.6 doesn't provide Qt5PlatformSupport.pc anymore
+    sed -i 's/Qt5PlatformSupport//' qt5-im/Makefile || return
```

Comment:
It’d be better to create a patch instead of doing 5 replacements in Makefile with sed.